.github/ISSUE_TEMPLATE/schema-changes-additions.md

```diff
@@ -0,0 +1,27 @@
+---
+name: "Schema Changes and Additions"
+about: "Changes or additions to the ECS schema"
```

Comment:
```suggestion
about: "Changes or additions to ECS"
```

The S in ECS already stands for schema ;-)

Reply:
🤦